akka-actor-tests/src/test/scala/akka/routing/MetricsBasedResizerSpec.scala

```diff
@@ -244,8 +244,8 @@ class MetricsBasedResizerSpec extends AkkaSpec(ResizerSpec.config) with DefaultT
 
       resizer.reportMessageCount(router.routees, router.msgs.size)
 
-      val after = LocalDateTime.now
-      resizer.performanceLog(2).toMillis shouldBe (java.time.Duration.between(before, after).toMillis / 2 +- 1)
+      val after = System.nanoTime()
+      resizer.performanceLog(2).toMillis shouldBe ((before - after).nanos.toMillis / 2 +- 1)
```

Comment:
performanceLog contains the average duration between two messages for a specific pool size (fixed to 2 in this case), so it passes two messages through, times that, and expects the duration to be half the time for two messages, give or take 1 millisecond.

Wait a minute, last time it failed it was with `0 was not 3 plus or minus 1`, so I guess that means time between first report and second was < 1 milli. I think I have to go back to thinking about this one.